src/config/machines/example-grant-machine.js

```diff
@@ -23,6 +22,20 @@ export const actionImplementations = {
     currentPageId: (_context, event) => {
       return event.type === 'NEXT' ? event.nextPageId : event.previousPageId;
     }
+  }),
+  updateAnswers: assign({
+    /**
+     * Updates the completed pages
```

Comment:
Is that what the function does? Maybe I don't understand these

Reply:
yes it just takes the args and defines the new object. It is not mutating the old it is returning new.